libbpf-tools/biolatency.bpf.c

```diff
@@ -6,9 +6,14 @@
 #include <bpf/bpf_tracing.h>
 #include "biolatency.h"
 #include "bits.bpf.h"
+#include <linux/version.h>
```

Comment:
please remove this include, we don't need it, do we?

Reply:
Remove this line got the following warining:
```
warning: implicit declaration of function 'KERNEL_VERSION' is invalid in C99 [-Wimplicit-function-declaration]
```

Reply:
ping this

Reply:
done